var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -60,6 +62,12 @@ class Vtk(CMakePackage):
     # VTK 8.1, that should change
     conflicts('+osmesa', when='+qt')
 
+    depends_on('python', when='+python')
+    depends_on('py-mpi4py', when='+mpi +python', type='run')
+    extends('python', when='+python')
+    # python3.7 compatibility patch backported from upstream
```

Comment:
@rubendibattista IMO it's worth including a link to the commit for a patch (e.g. for the author's comments). https://gitlab.kitware.com/vtk/vtk/commit/706f1b397df09a27ab8981ab9464547028d0c322 looks like the one.